manticore/core/smtlib/constraints.py

```diff
@@ -205,7 +205,7 @@ def new_bitvec(self, size, name='V', taint=frozenset()):
                          if not uniq a numeric nonce will be appended
             :return: a fresh BitVecVariable
         '''
-        assert size in (1, 4, 8, 16, 32, 64, 128, 160, 256)
+        assert (size == 1 or size % 8 == 0)
```

Comment:
Similar to below, since we're looking at this code anyway, could we port it over to use an Exception instead of an assert?